websites/I/IBL/dist/metadata.json

```diff
@@ -0,0 +1,55 @@
+{
+    "$schema": "https://schemas.premid.app/metadata/1.5",
+    "service": "Infinity Bots",
+    "author": {
+      "name": "Toxic Dev",
+      "id": "510065483693817867"
+    },
+    "contributors": [
+      {
+        "name": "Connor20024",
+        "id": "324646179134636043"
+      },
+      {
+        "name": "Anaxes",
+        "id": "567885938160697377"
+      },
+      {
+        "name": "Dexter",
+        "id": "780079091172900884"
+      }
+    ],

```

Comment:
Surely it doesn't take four people to make a 150 line presence

Reply:
Adjusted to myself and Anaxes